sqlserver/tests/conftest.py

```diff
@@ -68,13 +68,33 @@ def instance_docker():
     return deepcopy(INSTANCE_DOCKER)
 
 
+DEFAULT_TIMEOUT = 5
```

Comment:
should we set this even lower? e.g. 1 second? Waiting even 1 second to connect to the local instance seems reallyyy long

Reply:
Not sure. This is only for the integration tests so the goal is to minimize flakiness. It was 30 seconds before. I lowered it so the tests would fail more quickly. Do we need them to fail even more quickly? Not sure if 1 second may cause more flakiness 🤷‍♂️

Reply:
mmm okay. I wish we had more monitoring around this somehow. I started looking at the new CI visibility stuff https://docs.datadoghq.com/continuous_integration/explore_tests/ and im wondering if using that could give us some more insights into what is going on with these timeouts or just other flakey tests we own in general

Reply:
We have a dashboard here, at least shows what fails more often although it does not give detail https://p.datadoghq.com/sb/yB5yjZ-e9572aadd5162263629114009c09dce1